app/poros/movie.rb

```diff
@@ -1,16 +1,38 @@
-class Movie 
-  attr_reader :id, 
-              :title, 
-              :vote_average
-              
+class Movie
+  attr_reader :id,
+              :title,
+              :vote_average,
+              :runtime,
+              :overview,
+              :cast,
+              :reviews
+
   def initialize(movie_params)
-    @id = movie_params["id"]
-    @title = movie_params["title"]
-    @vote_average = movie_params["vote_average"]
+    @id = movie_params[:id]
+    @title = movie_params[:original_title]
+    @vote_average = movie_params[:vote_average]
+    @runtime = movie_params[:runtime]
+    @genres = movie_params[:genres]
+    @overview = movie_params[:overview]
+    @cast = movie_params[:cast].try(:first, 10)
+    @reviews = movie_params[:results]
+  end
+
+  #TODO: Consider Cast and Review poros 
+
+  #TODO: This isn't the place for this data parsing?
```

Comment:
This logic makes sense here to me.

Reply:
I still have some lingering doubts just because Meg highlighted using the facade for 'filtering' but this also made sense to me so I'm happy to keep it.